core/chains/evm/txmgr/orm.go

```diff
@@ -42,6 +42,78 @@ type orm struct {
 
 var _ ORM = (*orm)(nil)
 
+// Structs saved in database should be internal to
+// the ORM and distinct from generic params
+// so that modifying generic params does neccessitate
+// modification of underlying column types
+// (or runtime errors by mistake)
+type dbReceipt struct {
+	ID               int64
+	TxHash           common.Hash
+	BlockHash        common.Hash
+	BlockNumber      int64
+	TransactionIndex uint
+	Receipt          evmtypes.Receipt
+	CreatedAt        time.Time
+}
+
+type dbRawReceipt = evmtypes.Receipt
+
+type dbReceiptPlus struct {
+	ID           uuid.UUID        `db:"id"`
+	Receipt      evmtypes.Receipt `db:"receipt"`
+	FailOnRevert bool             `db:"FailOnRevert"`
```

Comment:
There is no `FailOnRevert` column

Reply:
It is coming from here: https://github.com/smartcontractkit/chainlink/blob/b05d42740d7037f8956ed02fbd0ce177f0587d80/core/chains/evm/txmgr/orm.go#L569-L571

Ultimately this corresponds to: https://github.com/smartcontractkit/chainlink/blob/b05d42740d7037f8956ed02fbd0ce177f0587d80/core/chains/evm/txmgr/models.go#L37

This design is 🤮

Reply:
Correct, no fail on revert column. Now, I don't know what the struct tags are used for or mean semantically but this is just based off [previously existing code](https://github.com/smartcontractkit/chainlink/commit/63b50f2cb09f86959da0e7e765ea4cc679d05557#diff-d6b76588977cc18e5b4c043aaae86416044e9b7e4233e6a79f5b05f30aa5219fR1497)

Reply:
Ah, I see, its a virtual column

Reply:
Can we rename these columns a bit? `id` -> `pipeline_run_id`, `FailOnRevert` -> `fail_on_revert`

Reply:
Yeah, I think it makes sense to rename them as private to the txmgr package. Why snake case though?